web/pages/post/[postId].tsx

```diff
@@ -84,6 +87,24 @@ function PostPage({
     onClose: onFormViewClose,
   } = useDisclosure();
 
+  useEffect(() => {
+
+    const logDogName = async () => {
+      const logger = getAnalyticsLogger();
+
+      // Log the click event
+      logger.logClickEvent({
+        objectId: `dog_${postData?.name}`,
+        userId: (Math.random() + 1).toString(36).substring(7),
+      });
+
+    }
+
+    if (postData?.name) {
+      logDogName().then().catch();
+    }
+  }, [JSON.stringify(postData)])
```

Comment:
Change dep array to just `postData`

Reply:
ngl my editor was giving me a weird error when I was doing this (but now its not for whatever reason) so this was a quick hack lol